samples/cellular/modem_shell/src/shell.c

```diff
@@ -144,6 +144,12 @@ int lwm2m_carrier_event_handler(const lwm2m_carrier_event_t *event)
 	case LWM2M_CARRIER_EVENT_REBOOT:
 		mosh_print("LwM2M carrier event: reboot");
 		break;
+	case LWM2M_CARRIER_EVENT_MODEM_DOMAIN:
+		mosh_print("LwM2M carrier event: modem domain");
```

Comment:
This is ok as such, but also the actual modem domain event could be printed here. It's a single integer.